src/library_async.js

```diff
@@ -828,6 +926,151 @@ mergeInto(LibraryManager.library, {
     });
   },
 
+  $Fibers__deps: ['$Asyncify'],
+  $Fibers: {
+    nextFiber: 0,
```

Comment:
null (or undefined) maybe?

Reply:
Is there a reason for null over 0? This holds C pointers rather than JS objects, so I thought 0 is appropriate. I don't feel strongly about it in any way, just curious.

Reply:
I saw `Fibers.nextFiber = null;`  below and assumed it was an object.  If its a pointer then zero is fine but it should be the same below I guess?

Reply:
Ah yes, the `Fibers.nextFiber = null;` is a mistake. Agreed it should be consistent.